MountRoulette.lua

```diff
@@ -60,17 +60,19 @@ function update_allowed_mounts()
 
     for _, id in ipairs(kis) do
         local ki = resources.key_items[id]
-        if ki.category == 'Mounts' and ki.name ~= "trainer's whistle" then -- Don't care about the quest KI
-            local mount_index = possible_mounts:find(function(possible_mount)
-                return windower.wc_match(ki.name:lower(), '♪' .. possible_mount .. '*')
-            end)
-            local mount = possible_mounts[mount_index]
-
-            -- Add this to allowed mounts if it is not already there and it is not blacklisted
-            if not allowed_mounts:contains(mount) and not S(blacklist):contains(mount) then
-                allowed_mounts_set:add(mount)
-            end
-        end
+        if (ki ~= nil) then
+			if ki.category == 'Mounts' and ki.name ~= "trainer's whistle" then -- Don't care about the quest KI
+				local mount_index = possible_mounts:find(function(possible_mount)
+					return windower.wc_match(ki.name:lower(), '♪' .. possible_mount .. '*')
+				end)
+				local mount = possible_mounts[mount_index]
+
+				-- Add this to allowed mounts if it is not already there and it is not blacklisted
+				if not allowed_mounts:contains(mount) and not S(blacklist):contains(mount) then
+					allowed_mounts_set:add(mount)
+				end
+			end
+		end
```

Comment:
```suggestion
        if (ki ~= nil) then
            if ki.category == 'Mounts' and ki.name ~= "trainer's whistle" then -- Don't care about the quest KI
                local mount_index = possible_mounts:find(function(possible_mount)
                    return windower.wc_match(ki.name:lower(), '♪' .. possible_mount .. '*')
                end)
                local mount = possible_mounts[mount_index]

                -- Add this to allowed mounts if it is not already there and it is not blacklisted
                if not allowed_mounts:contains(mount) and not S(blacklist):contains(mount) then
                    allowed_mounts_set:add(mount)
                end
            end
        end
```
In order to get this merged to the Windower project, the indentation needs to be in 4x spaces rather than tabs.